csharp/src/Google.Protobuf/Reflection/EnumDescriptor.cs

```diff
@@ -127,6 +128,26 @@ public EnumValueDescriptor FindValueByName(string name)
         /// <summary>
         /// The (possibly empty) set of custom options for this enum.
         /// </summary>
-        public CustomOptions CustomOptions => Proto.Options?.CustomOptions ?? CustomOptions.Empty;
+        //[Obsolete("CustomOptions are obsolete. Use GetOption")]
```

Comment:
> This change added extension and will Obsolete custom options. 

It does not actually obsolete custom options yet, @anandolee, since I can't add the necessary APIs to replace custom options until the descriptor protos have been made public. Originally it obsoleted custom options but now it does not due to [this review](https://github.com/protocolbuffers/protobuf/pull/5350#discussion_r239288615).

Reply:
Ack. Thanks for make it clear